src/containers/Home/dashboardHome.tsx

```diff
@@ -59,7 +59,7 @@ export const DashboardHome = () => {
                             {UIConstants.teamMemberTitle}
                         </Typography>
                     </AccordionSummary>
-                    <AccordionDetails>test 1</AccordionDetails>
+                    <AccordionDetails>{CardBoxList}</AccordionDetails>
```

Comment:
CardBoxList is a component. Pls import it like <CardBoxList />

Reply:
Done